src/PortAudio.jl

```diff
@@ -135,25 +143,22 @@ Audio devices can either be `PortAudioDevice` instances as returned
 by `PortAudio.devices()`, or strings with the device name as reported by the
 operating system. If a single `duplexdevice` is given it will be used for both
 input and output. If no devices are given the system default devices will be
-used.
+used. Over- and under-run messages will be sent to the debug log.
 
 Options:
 
 * `eltype`:         Sample type of the audio stream (defaults to Float32)
 * `samplerate`:     Sample rate (defaults to device sample rate)
 * `latency`:        Requested latency. Stream could underrun when too low, consider
                     using provided device defaults
-* `warn_xruns`:     Display a warning if there is a stream overrun or underrun, which
```

Comment:
I thought this issue was about suppressing the start up messages. Is it also about removing functionality? It's easier to review and maintain a project if the PR description matches the code and if each PR makes a single change.

I think warning about stream overruns may be important to keep. And if it's moved to the debug users may miss it. If I was running an experiment and the buffer is over/under running I would want to be warned immediately.

This is different to suppressing the initial startup messages, as they may falsely give the impression something is wrong. Whereas these messages do imply something may be going wrong, not a debug issue.

Please correct me if I have misunderstood here.

Reply:
And I dont feel bad for having accidentaly pinged those users with awesome handles.

Reply:
Sure, I can move this to a separate PR or discard it. It's slightly breaking and doesn't add new functionality. I just thought these warning messages would be a good fit for the debug log because they aren't shown by default but only if the user asks for them. My understanding is that xruns happen at least every GC cycle, so pretty much all the time.

Reply:
If I were getting xruns on every GC I would increase the latency or reduce the amount of garbage I'm generating. Generally xruns are accompanied by audible clicks in output or dropouts in input, so if you're doing anything important (like lab measurements) I would definitely consider them errors. For casual users it might not be a big deal, but also they might not care so much about latency so it's probably a good idea to increase the latency in that case.

I'm not actually convinced that hiding xrun notifications is the right default, but I think we can separate that out into a separate issue/PR.